app/assets/stylesheets/exception_hunter/errors.css

```diff
@@ -94,12 +103,31 @@
     overflow-x: hidden;
 }
 
+.error-cell__tags {
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  padding-bottom: 3px;
+}
+.error-cell__tags span {
```

Comment:
You shouldn't reference HTML elements in css directly except for special occasions. Add a class like `.error-cell__tags .error-tag`